src/components/utils/rqb-inputs/value-editor.tsx

```diff
@@ -38,10 +51,12 @@ const ValueEditor = (props: ValueEditorProps) => {
         return <TextValueEditor {...props} />;
     }
     return (
-        <MaterialValueEditor
-            {...props}
-            title={undefined} // disable the tooltip
-        />
+        <Box sx={props.inputType === 'number' ? styles.noArrows : undefined}>
```

Comment:
I think that we don't need to check for input type and that we can apply the style in any case.

Reply:
I'd rather not add unecessary styling that could be confusing down the line.